README.md

```diff
@@ -1 +0,0 @@
-# azure-storage-samples-for-python
```

Comment:
Instead of deleting this README you could add links in it to the other READMEs

Reply:
Yes, would be great to explain layout of directories here